pkgs/development/interpreters/python/hooks/python-shebangs-fixup-hook.sh

```diff
@@ -0,0 +1,24 @@
+# shellcheck shell=bash
+
+# Setup hook for fixing Python shebangs
```

Comment:
Ok lastly I would just put a little more explanation as to why this is needed. My guess is it's something like "when cross-compiling patchShebangs patches with the wrong Python shebang so we need to fix it up" 

Other than that lgtm.